TESTS/usb_device/basic/USBEndpointTester.cpp

```diff
@@ -39,10 +37,10 @@
 
 #define VENDOR_TEST_CTRL_IN             1
 #define VENDOR_TEST_CTRL_OUT            2
-#define VENDOR_TEST_CTRL_IN_SIZES       9
-#define VENDOR_TEST_CTRL_OUT_SIZES      10
-#define VENDOR_TEST_RW_RESTART          11
-#define VENDOR_TEST_ABORT_BUFF_CHECK    12
+#define VENDOR_TEST_CTRL_IN_SIZES       3
+#define VENDOR_TEST_CTRL_OUT_SIZES      4
+#define VENDOR_TEST_RW_RESTART          5
+#define VENDOR_TEST_ABORT_BUFF_CHECK    6
```

Comment:
Can you put in a header so there is no duplication between this source file and USBTester.cpp